### ISSUE TITLE
Image on /earn is missing alt text

### ISSUE THREAD

**Issue by youbrokesomething** [Automattic/wp-calypso#75603]
### Quick summary

An image on /earn is missing alt text. The image is `<img src="/calypso/images/../images/earn-section-c355053dfc3f1be38946.svg">`. If the image isn't important to the page's context, it can have an empty alt tag or a "presentation" role.

[Here's a guide to help decide what tag to use.](https://www.w3.org/WAI/tutorials/images/decision-tree/#:~:text=wasn't%20there%3F-,Yes%3A,the%20link%20or%20action%20taken)

### Steps to reproduce

1. On /earn
2. Inspect the image referenced in the summary or use an accessibility checker like the axe DevTools Chrome extension to scan the page.

### What you expected to happen

All image should have an alt tag or a role that excludes the image from screen readers.

### What actually happened

At least one image is missing a tag or role.

### Impact

Some (< 50%)

### Available workarounds?

No but the platform is still usable
